plugins/generic/externalFeed/ExternalFeedPlugin.inc.php

```diff
@@ -1,375 +1,329 @@
-						$externalFeedForm->readInputData();
-					} else {
-						$externalFeedForm->initData();
-					}
-					$templateMgr->assign('journalSettings', $journalSettings);
-					$externalFeedForm->display();
-				} else {
-					$request->redirect(null, 'manager', 'plugin', array('generic', $this->getName(), 'feeds'));
-				}
-				return true;
-			case 'update':
-				$externalFeedId = $request->getUserVar('feedId') == null ? null : (int) $request->getUserVar('feedId');
-				$externalFeedDao = DAORegistry::getDAO('ExternalFeedDAO');
-
-				if (($externalFeedId != null && $externalFeedDao->getExternalFeedJournalId($externalFeedId) == $journalId) || $externalFeedId == null) {
-
-					$this->import('ExternalFeedForm');
-					$externalFeedForm = new ExternalFeedForm($this, $externalFeedId, $journalId);
-					$externalFeedForm->readInputData();
-
-					if ($externalFeedForm->validate()) {
-						$externalFeedForm->execute();
-
-						if ($request->getUserVar('createAnother')) {
-							$request->redirect(null, 'manager', 'plugin', array('generic', $this->getName(), 'create'));
-						} else {
-							$request->redirect(null, 'manager', 'plugin', array('generic', $this->getName(), 'feeds'));
-						}
-					} else {
-						if ($externalFeedId == null) {
-							$templateMgr->assign('externalFeedTitle', 'plugins.generic.externalFeed.manager.createTitle');
-						} else {
-							$templateMgr->assign('externalFeedTitle', 'plugins.generic.externalFeed.manager.editTitle');
-						}
-
-						$journalSettingsDao = DAORegistry::getDAO('JournalSettingsDAO');
-						$journalSettings =& $journalSettingsDao->getSettings($journalId);
-
-						$templateMgr->assign('journalSettings', $journalSettings);
-						$externalFeedForm->display();
-					}
-				} else {
-					$request->redirect(null, 'manager', 'plugin', array('generic', $this->getName(), 'feeds'));
-				}
-				return true;
-			case 'settings':
-				$this->import('ExternalFeedSettingsForm');
-				$form = new ExternalFeedSettingsForm($this, $journal->getId());
-				if ($request->getUserVar('save')) {
-					$request->redirect(null, 'manager', 'plugin', array('generic', $this->getName(), 'feeds'));
-				} elseif ($request->getUserVar('uploadStyleSheet')) {
-					$form->uploadStyleSheet();
-				} elseif ($request->getUserVar('deleteStyleSheet')) {
-					$form->deleteStyleSheet();
-				}
-				$form->initData();
-				$form->display();
-				return true;
-			case 'feeds':
-			default:
-				$this->import('ExternalFeed');
-				$rangeInfo = Handler::getRangeInfo($request, 'feeds');
-				$externalFeedDao = DAORegistry::getDAO('ExternalFeedDAO');
-				$feeds =& $externalFeedDao->getExternalFeedsByJournalId($journalId, $rangeInfo);
-				$templateMgr->assign('feeds', $feeds);
-
-				$templateMgr->display($this->getTemplatePath() . 'externalFeeds.tpl');
-				return true;
-		}
-	}
-}
-
-?>
+<?php
+
+/**
+ * @file plugins/generic/externalFeed/ExternalFeedPlugin.inc.php
+ *
+ * Copyright (c) 2013-2018 Simon Fraser University Library

```

Comment:
There are still some copyrights referring to Simon Fraser University Library, not Simon Fraser University.